beets/autotag/hooks.py

```diff
@@ -609,25 +619,17 @@ def album_candidates(items, artist, album, va_likely, extra_tags):
     constrain the search.
     """
 
+    common_args = [album, len(items), extra_tags]
```

Comment:
I know this might be bikeshedding, but I actually think the repetition (passing the same set of arguments to the two calls to `mb.match_album`) is a little more intelligible.